ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/ValidatedCommandFactory.java

```diff
@@ -105,9 +108,31 @@ private static Command createCommand(
       return createForTerminateQuery(statement, context);
     }
 
+    if (statement.getStatement() instanceof AlterSystemProperty) {
+      return createForAlterSystemQuery(statement, context);
+    }
+
     return createForPlannedQuery(statement, serviceContext, context);
```

Comment:
Do you mean updating the `statement` for `createForPlannedQuery` or for `createForAlterSystemQuery`? Because I am not using the config in my validation?